core/tools/bluez_emu_bridge/device1.py

```diff
@@ -0,0 +1,319 @@
+    def Modalias(self) -> "s":
+        return "usb:v1D6Bp0246d054F"
+
+    @dbus_property(access=PropertyAccess.READ)
+    def Name(self) -> "s":
+        return self._name
+
+    @dbus_property(access=PropertyAccess.READ)
+    def Paired(self) -> "b":
+        return self.is_bonded()
+
+    @dbus_property(access=PropertyAccess.READ)
+    def RSSI(self) -> "n":
+        return self._rssi
+
+    @dbus_property(access=PropertyAccess.READ)
+    def ServicesResolved(self) -> "b":
+        return self._services_resolved
+
+    @dbus_property(access=PropertyAccess.READWRITE)
+    def Trusted(self) -> "b":
+        return True
+
+    @Trusted.setter
+    def Trusted(self, value: "b"):
+        LOG.debug(f"Trying to set Trusted to {value}")
+
+    @dbus_property(access=PropertyAccess.READ)
+    def TxPower(self) -> "n":
+        return 7
+
+    @dbus_property(access=PropertyAccess.READ)
+    def UUIDs(self) -> "as":
+        uuids = []
+        for srv in self._services:
+            uuids.append(srv._uuid)
+            for chr in srv._characteristics:
+                uuids.append(chr._uuid)
+        return uuids
+
+    async def _update_connected(self, new_value: bool):
+        await asyncio.sleep(random.uniform(0.5, 1.5))
+        property_changed = {"Connected": new_value}
+        self.emit_properties_changed(property_changed)
+        LOG.debug(f"Property changed: {property_changed}")
+
+    async def _update_services_resolved(self, new_value: bool):
+        await asyncio.sleep(random.uniform(0.0, 0.5))
+        self._services_resolved = new_value
+        property_changed = {"ServicesResolved": self._services_resolved}
+        self.emit_properties_changed(property_changed)
+        LOG.debug(f"Property changed: {property_changed}")
+
+    async def _update_paired(self, new_value: bool):
+        property_changed = {"Paired": new_value}
+        self.emit_properties_changed(property_changed)
+        LOG.debug(f"Property changed: {property_changed}")
+
+    async def _update_rssi(self, new_value: int):
+        self._rssi = int(new_value)
+        property_changed = {"RSSI": self._rssi}
+        self.emit_properties_changed(property_changed)
+
+    async def connection_state_task(self, write_fn, queue):
+        self.send_event_fn = write_fn
+        self.command_queue = queue
+        self.send_event(Event.ping())
+        while True:
+            command = await queue.get()
+            LOG.debug(f"Emulator sent command: {command}")
+            self.handle_command(command)
+
+    def handle_command(self, command):
+        # handle new status from mock driver
+        command = Command.parse(command)
+
+        LOG.debug(f"Command {command}")
+        t = command.command_type
+        if t == CommandType.STATUS:
+            pass
+        elif t == CommandType.PAIRING_MODE:
+            pass
+        elif t == CommandType.DISCONNECT:
+            LOG.error(f"Command not implemented: {command}")
```

Comment:
nit: this happens when calling `trezorctl device wipe` - how should it be handled?

(not urgent - can be definitely done in a separate PR)